include/libopencm3/stm32/g0/adc.h

```diff
@@ -37,7 +37,8 @@
 
 /** @defgroup adc_reg_base ADC register base addresses
  *@{*/
-#define ADC1				ADC1_BASE
+#define ADC1			ADC1_BASE
+#define ADC			ADC1_BASE /* for API compatibility */
```

Comment:
I'd rather not do this, I'd rather always have the 1, what are you trying to be compatible with here?  What doesn't accept the ADC1 form?

Reply:
I was relying on this: https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/f0/adc.h#L47